latest.py

```diff
@@ -1,134 +1,60 @@
-#----------------------------------------------------------------------
-#   
-#   Project:            New goto funtion
-#   Description:        Just point the target coordinates
-#                       
-#   Date:               03.12.2021
-#   Maximum score:      70 points
-#   Time left:          31 seconds
-#
-#   Starting position:  A
-#   Orientation:        Facing East (standard, 0 for the robot)
-#   Preload:            Zero Rings
```

Comment:
Thanks for shortening

Reply:
Please extend later in the updates of latest.py